kafka/src/test/java/org/axonframework/extensions/kafka/eventhandling/producer/DefaultProducerFactoryTests.java

```diff
@@ -117,9 +117,10 @@ public void testSendingMessages_UsingMultipleProducers() throws ExecutionExcepti
         List<Producer<String, String>> producers = new ArrayList<>();
         List<Future<RecordMetadata>> results = new ArrayList<>();
         String topic = "testSendingMessagesUsingMultipleProducers";
-        for (int i = 0; i < 10; i++) {
+        for (int i = 0; i < 12; i++) {
```

Comment:
Do you care to elaborate why you've changed the test case like this @lbilger?

Reply:
The problem I tried to fix here actually happened on the 12th event sent through the producer:
- `producerCacheSize` default value is 10
- after sending the 11th event, the producer is closed because the cache size is exceeded
- the 12th event fails to send because the producer is closed
This change made the test fail in the original code so I had a failing test to work on.

Reply:
I suggest to get the default value from the properties, add two or provide a comment for this magic number. To make the test more robust, I'll suggest to test relative to the default value.

Reply:
Good idea, but there seems to be no API to get the default value and I wouldn't want to introduce one just for the sake of this test. I'll add a comment instead.

Reply:
Awesome, thanks for elaborating @lbilger.